### ISSUE TITLE
Scientific Calculator functions?

### ISSUE THREAD

**Issue by Gandalf-the-Blue** [FossifyOrg/Calculator#24]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calculator/issues) or [closed](https://github.com/FossifyOrg/Calculator/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calculator/discussions) or [closed](https://github.com/FossifyOrg/Calculator/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calculator/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Add scientific calculator functions such as square, root, log, trignometry etc

### Why do you want this feature?

Pretty useful for calculations

**Comment by fohfuu** [comment]
Agreed. Android has had these in the default calculator apps for a long time, in [Calculator](https://android.googlesource.com/platform/packages/apps/Calculator/) and then [Exact Calculator](https://android.googlesource.com/platform/packages/apps/ExactCalculator/).

**Comment by JarrColl** [comment]
Especially PI for calculating circumference, area of a circle, etc would be useful.

**Comment by IsHacker003** [comment]
Exponentiation (^) and square root functions are already there, but trigonometry and pi hasn't been added.

**Comment by my-garbage** [comment]
<img alt="ignoreImageMinify" src="https://github.com/user-attachments/assets/d7137c48-f61d-489c-84dd-bb92ee8c7619" width=188 />

Splitting into a business and a scientific calculator mode would help. Because not everybody needs scientific calculation.
And I absolutely agree that every student totally needs the functions requested.
